### ISSUE TITLE
Cannot download files from Jupyter notebook

### ISSUE THREAD

**Issue by BengtHolm** [ChandraLingam/AmazonSageMakerCourse#5]
Dear Chandra

In order to try out your newest additions to the xgboost material,
I made a git stash, and then a git pull command in the AmazonSageMakerCourse directory.
However, now, every time I try do download a file, I get the error

Jupyter Notebook
403 : Forbidden
The error was:

Blocking Cross Origin request from https://bengtsdemo1.notebook.us-east-1.sagemaker.aws/tree/AmazonSageMakerCourse/fm.

So basically I cannot download a file to my local computer.
Any Ideas?

Regards, Bengt Holm

**Comment by ChandraLingam** [comment]
Could you please try from the command line terminal as shown in the house keeping lecture?  (it appears you are trying from notebook page). Please confirm where you ran the git pull command.
